rocAL/include/loaders/image/image_loader.h

```diff
@@ -55,6 +55,7 @@ class ImageLoader : public LoaderModule {
     void shut_down() override;
     void feed_external_input(const std::vector<std::string>& input_images_names, const std::vector<unsigned char*>& input_buffer,
                              const std::vector<ROIxywh>& roi_xywh, unsigned int max_width, unsigned int max_height, unsigned int channels, ExternalSourceFileMode mode, bool eos) override;
+    size_t last_batch_padded_size() override;
```

Comment:
why is this needed for external source reader?

Reply:
This is added to the image loader. We haven't added last batch policy to external source reader since the user would be passing the filenames/buffers for each epoch